smithy-aws-traits/src/test/resources/software/amazon/smithy/aws/traits/diffs/sigv4a-before-sigv4-migration.events

```diff
@@ -0,0 +1 @@
+[DANGER] ns.foo#Service: The `aws.auth#sigv4a` authentication scheme was added before the `aws.auth#sigv4` authentication scheme in the effective auth schemes for `ns.foo#Service`. The `aws.auth#sigv4a` authentication scheme cannot be added before an existing `aws.auth#sigv4` authentication scheme since not all credentials usable by `aws.auth#sigv4` are compatible with `aws.auth#sigv4a`, and can break existing clients' authentication. Make trait changes (`@auth`) so that `aws.auth#sigv4a` is added after `aws.auth#sigv4` in the effective auth schemes, and ensure authentication succeeds with clients built from both the current and new models. | SigV4Migration
```

Comment:
> Make trait changes (`@auth`) so that `aws.auth#sigv4a` is added after `aws.auth#sigv4` in the effective auth schemes

Do we want to say this? Per SRA, adding sigv4a after sigv4 seems like a no-op, since sigv4 will always be picked and never sigv4a. I think (possibly incorrectly) services that want to add sigv4a want to add it as the preferred one, but we just want them to evaluate the risk for their customers for the edge cases, and suppress this event, if they still want to move forward. Suggesting them to move sigv4a after sigv4 probably doesn't solve their use case. Either way I think we don't need to be prescriptive about the solution (though suggestions can be good), but make them aware of the consequences.